tools/flaky-test-reporter/github_issue.go

```diff
@@ -0,0 +1,460 @@
+/*
```

Comment:
As a general comment, I miss comments identifying large blocks of code. e.g., something like "// close open issue if the test passed" on line 195.

Reply:
good idea, added